src/packagedcode/conda.py

```diff
@@ -51,6 +51,7 @@ def parse(cls, location, package_only=False):
                 primary_language=cls.default_primary_language,
                 dependencies=dependencies,
                 extra_data=extra_data,
+                is_private=True,
```

Comment:
Why private?

Reply:
Conda environment YAMLs are ostly used to export a particular conda environment (this is like a lockfile used in projects) and to import the environment from them to replicate the conda environment. See https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file and https://docs.conda.io/projects/conda/en/stable/user-guide/concepts/environments.html
This is not a manifest that you can publish a conda package, for that you need build recipes in meta.yaml files, since this manifest is not used to publish packages at all, and only publish environments, I've marked the packages created as private.